repositories.yaml

```diff
@@ -164,10 +164,11 @@ logins-store:
   notification_emails:
     - rfkelly@mozilla.com
     - lina@mozilla.com
-    - application-services@mozilla.com
+    - sync-team@mozilla.com
```

Comment:
We don't really use the `application-services@` email address any more, so I took the liberty of updating it while I was in here.